components/datetime/src/format/datetime.rs

```diff
@@ -120,7 +120,7 @@ where
     Ok(())
 }
 
-fn write_field<T, W>(
+pub(super) fn write_field<T, W>(
```

Comment:
These need to become `pub(super)` because I put `datetime`, `timezone`, and `zoned_datetime` into separate file modules.